### ISSUE TITLE
Desktop landing page has mobile view

### ISSUE THREAD

**Issue by twentysixmoons** [open-austin/budgetparty-landing#7]
@mateoclarke said it's probably some conflicting CSS. 

EX:
<img width="757" alt="screen shot 2017-06-30 at 5 41 06 pm" src="https://user-images.githubusercontent.com/10767430/27756230-69314dfc-5dbb-11e7-8491-2c25a57159d5.png">

**Comment by mateoclarke** [comment]
Yes, since the "/home" route gets loaded in as a child component, it inherits the width constraint we put on the rest of the app via the `.container` class. Its icky.
